ui/lib/core/addon/components/confirmation-modal.hbs

```diff
@@ -21,21 +13,21 @@
         class="input has-margin-top"
         autocomplete="off"
         spellcheck="false"
-        data-test-confirmation-modal-input={{this.testSelector}}
+        data-test-confirmation-modal-input={{@testSelector}}
       />
     </div>
   </section>
   <footer class="modal-card-foot modal-card-foot-outlined">
     <button
       type="button"
-      class="button {{this.buttonClass}}"
+      class={{concat "button " this.buttonClass}}
       disabled={{not-eq this.confirmationInput this.confirmText}}
-      onclick={{this.onConfirm}}
-      data-test-confirm-button={{this.testSelector}}
+      {{on "click" @onConfirm}}
+      data-test-confirm-button={{@testSelector}}
     >
       {{this.buttonText}}
     </button>
-    <button type="button" class="button is-secondary" onclick={{action (mut this.isActive) false}} data-test-cancel-button>
```

Comment:
I carried this over to mimic how the `Modal` works. Since the passed in `onClose` action changes `isActive` to `false`, I thought it'd be better to keep this consistent and handle changing `isActive` in only one place - the function passed in. What do you think?

in `database-connection.hbs`:

```
<ConfirmationModal
  @title="Delete connection?"
  @onClose={{action (mut this.isDeleteModalActive) false}}
  @isActive={{this.isDeleteModalActive}}
  @confirmText={{@model.name}}
  @toConfirmMsg="deleting the connection"
  @onConfirm={{action "delete"}}
  @testSelector="delete"
>
```